go.mod

```diff
@@ -11,9 +11,11 @@ require (
 	github.com/free5gc/ngap v1.1.1
 	github.com/free5gc/openapi v1.2.2
 	github.com/free5gc/util v1.3.0
+	github.com/google/gopacket v1.1.20-0.20220810144506-32ee38206866
 	github.com/ishidawataru/sctp v0.0.0-20250303034628-ecf9ed6df987
 	github.com/songgao/water v0.0.0-20200317203138-2b4b6d7c09d8
 	github.com/spf13/cobra v1.10.1
+	github.com/tiagomdiogo/ScaGo v0.0.0-20250909151929-0aed579be48e
```

Comment:
This is a pretty niche dependency, i.e. it's not super well maintained nor used. At least it doesn't depend on a bunch of other things itself and we don't include it in Ella Core.